src/components/Compare/CompareMain.tsx

```diff
@@ -175,15 +163,16 @@ const CompareMain = (props: {
     scrollToCompare();
   };
 
-  // Location page slider:
-  const defaultSliderValue = scopeValueMap[geoScope];
-  const [sliderValue, setSliderValue] = useState(defaultSliderValue);
-
-  // Homepage slider:
-  const defaultHomepageSliderValue = homepageScopeValueMap[homepageScope];
-  const [homepageSliderValue, setHomepageSliderValue] = useState(
-    defaultHomepageSliderValue,
-  );
+  // Since the route isn't changing when navigating between county pages within the same state, state variables weren't resetting. This forces a reset:
```

Comment:
This useEffect is already in develop, but **this current** PR's branch is based off of the branch where the effect was added, so the change is showing up here as well